ImaginaryTests-iOS/TestHelper.swift

```diff
@@ -14,3 +15,13 @@ class TestHelper {
     return image!
   }
 }
+
+class MockDownloader: ImageDownloader {
```

Comment:
private?

Reply:
@vadymmarkov It is used by many tests, so let's keep it `internal`